packages/editor-ui/src/components/ResourceMapper/ResourceMapper.vue

```diff
@@ -354,11 +355,14 @@ async function loadAndSetFieldsToMap(): Promise<void> {
 			}
 			return field;
 		});
-		state.paramValue = {
-			...state.paramValue,
-			schema: newSchema,
-		};
-		emitValueChanged();
+
+		if (!isEqual(newSchema, state.paramValue.schema)) {
+			state.paramValue = {
+				...state.paramValue,
+				schema: newSchema,
+			};
```

Comment:
Mutate the state instead of replacing it
```suggestion
			state.paramValue.schema = newSchema
```